doc/source/examples/multiphysics/sloshing-in-rectangular-tank/sloshing-in-rectangular-tank.rst

```diff
@@ -0,0 +1,198 @@
+================================
+Sloshing in a rectangular tank
+================================
+
+This example simulates the damping of a small amplitude wave for Reynolds number of (2, 20, 200 and 2000). The problem is inspired by the test case of `Carrica et al. [1] <https://onlinelibrary.wiley.com/doi/abs/10.1002/fld.1279>`_. 
+
+
+----------------------------------
+Features
+----------------------------------
+- Solver: ``gls_navier_stokes_2d`` 
+- Volume of fluid (VOF)
+- Unsteady problem handled by an adaptive BDF2 time-stepping scheme 
+- Usage of a python script for post-processing data
+
+---------------------------
+Files used in this example
+---------------------------
+- ``examples/multiphysics/sloshing-in-rectangular-tank/sloshing_0002/sloshing-in-rectangular-tank_Re0002.prm``
+- ``examples/multiphysics/sloshing-in-rectangular-tank/sloshing_0020/sloshing-in-rectangular-tank_Re0020.prm``
+- ``examples/multiphysics/sloshing-in-rectangular-tank/sloshing_0200/sloshing-in-rectangular-tank_Re0200.prm``
+- ``examples/multiphysics/sloshing-in-rectangular-tank/sloshing_2000/sloshing-in-rectangular-tank_Re2000.prm``
+
+-----------------------------
+Description of the case
+-----------------------------
+
+Predicting the dynamics of free surface waves is essential for many industrial applications (e.g. transport of liquified natural gas). Yet, simulating their dynamics is difficult, especially for high Reynolds number values . Indeed, in this case, the amplitude of the waves dampen very slowly. This leads to an oscillatory wave problem which is highly sensitive to the time integration scheme and the coupling between the VOF solver and the Navier-Stokes solver. 
+
+In this problem, we simulate the damping of a small amplitude wave in a rectangular cavity defined from  :math:`(-1,-1)' to :math:`(1,0.1)`. The initial height of the wave :math:`\xi (x)` is given by:
```

Comment:
I left  typo in my earlier suggestion (thanks to @AmishgaAlphonius for pointing it out!)
```suggestion
In this problem, we simulate the damping of a small amplitude wave in a rectangular cavity defined from  :math:`(-1,-1)` to :math:`(1,0.1)`. The initial height of the wave :math:`\xi (x)` is given by:
```